breacharbiter.go

```diff
@@ -270,12 +255,14 @@ func (b *breachArbiter) Start() error {
 
 	// Spawn the exactRetribution tasks to monitor and resolve any breaches
 	// that were loaded from the retribution store.
-	for chanPoint, closeSummary := range closeSummaries {
+	for chanPoint := range breachRetInfos {
+		retInfo := breachRetInfos[chanPoint]
+
 		// Register for a notification when the breach transaction is
 		// confirmed on chain.
-		breachTXID := closeSummary.ClosingTXID
+		breachTXID := retInfo.commitHash
 		confChan, err := b.cfg.Notifier.RegisterConfirmationsNtfn(
-			&breachTXID, 1, uint32(currentHeight))
+			&breachTXID, 1, uint32(currentHeight-6))
```

Comment:
Is the `-6` arbitrary? :b

Reply:
was just an arbitrary safety buffer, will remove :P

Reply:
For that height hint we should actually pass in either: the height the funding transaction was confirmed, or when the channel was closed on chain. The latter is the more correct of the two. We need to do this as otherwise, we may miss a historical conf dispatch if in neutrino mode.